test/busted_runner.lua

```diff
@@ -9,4 +9,12 @@ if platform and platform.sysname:lower():find'windows' then
   end
 end
 
+local deps_install_dir = os.getenv 'DEPS_INSTALL_DIR'
+package.path = deps_install_dir.."/share/lua/5.1/?.lua;"..deps_install_dir.."/share/lua/5.1/?/init.lua;"..package.path
```

Comment:
is the `deps_prefix.."/share/lua/5.1/?.lua;"..deps_prefix` line also needed togheter with this or can it now be removed?

Reply:
Nice catch. The first three `deps_prefix` should be safe to remove as those directories will never be created in the first place. Unsure about the last one (`deps_prefix.."/bin/?.dll`), I'll test removing it to see if anything breaks.